src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,29 @@
+<%--
+  admin.jsp
+  Alana Dillinger
+--%>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Administration</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+<%@ include file="../component/navbar.jsp" %>
+
+  <div id="container">
+    <div
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+
+      <h1>This will be the Administration page</h1>
+
+      <% if(request.getAttribute("error") != null){ %>
```

Comment:
Nice error handling! :) Error handling tend to be forgotten when iterating fast. But having good error handling can give users more confidence in the product than showing stack traces of error. This is a fun one: https://i.pinimg.com/originals/d4/f1/e0/d4f1e063783c76f4daf94b4a27a2d496.png
:P